### DISCUSSION TITLE
Cannot even launch Cowsay from Git CMD

### DISCUSSION THREAD

**Discussion by DaniyarNauryz** [community/community#22091] [New to GitHub]
<p>Stuck at the second step, trying to launch <a href="https://github.com/schacon/cowsay" rel="nofollow noopener">https://github.com/schacon/cowsay</a>. Screenshot below. Any suggestions? The same applies to <a href="https://github.com/scslab/tcpcrypt" rel="nofollow noopener">https://github.com/scslab/tcpcrypt</a>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/99f1a13c0b2f6a23722570e175649f83957dbd73.png" data-download-href="/uploads/short-url/lXQK278FkRF5flThXdmhQkxQBLd.png?dl=1" title="cowsay daniyar" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/68249290/181068844-d3377ffd-be9f-42e2-9fa0-27c7b8313820.png" alt="cowsay daniyar" data-base62-sha1="lXQK278FkRF5flThXdmhQkxQBLd" width="690" height="319" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/99f1a13c0b2f6a23722570e175649f83957dbd73_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">cowsay daniyar</span><span class="informations">978×453 5.62 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tmetz** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>To use Linux commands like <code>ls</code> you’ll want to install and launch Git Bash instead of Git CMD.  You should see an option to install Git Bash during the installation process for <a href="https://gitforwindows.org/">Git for Windows</a>.</p>

**Comment by DaniyarNauryz** [comment]
<blockquote>
<p>Blockquote</p>
</blockquote>
<p>Hi there! :wave: Welcome to the Community!</p>
<p>To use Linux commands like  <code>ls</code>  you’ll want to install and launch Git Bash instead of Git CMD. You should see an option to install Git Bash during the installation process for <a href="https://gitforwindows.org/" rel="nofollow noopener">Git for Windows</a>.</p>
<blockquote>
<p>Blockquote</p>
</blockquote>
<p>Nice! Very simple and effective!</p>
